pkgs/build-support/fetchzip/default.nix

```diff
@@ -28,7 +28,7 @@ lib.overrideDerivation (fetchurl ({
       mkdir "$unpackDir"
       cd "$unpackDir"
 
-      renamed="$TMPDIR/$name"
+      renamed="$TMPDIR/${baseNameOf url}"
```

Comment:
Does this change belong in this PR? It seems like it'll cause quite a few things to rebuild.